source/install/ee-install.rst

```diff
@@ -4,43 +4,46 @@
 Enterprise Install and Upgrade
 ===========================================
 
-Mattermost Enterprise Edition is free to use in "team mode" without enterprise features enabled. Enable enterprise features using a `trial license <https://about.mattermost.com/trial/>`__ or by `purchasing a license key <https://about.mattermost.com/pricing/>`__. There are two variants of Enterprise Edition, E10 and E20, and you can `compare their features online <https://mattermost.com/pricing-feature-comparison/>`__.
+Mattermost Enterprise Edition is free to use in "team mode" without enterprise features enabled. There are two variants of Enterprise Edition, E10 and E20, and you can `compare their features online <https://mattermost.com/pricing-feature-comparison/>`__.
+
+Once you've downloaded and installed Mattermost, start an Enterprise E20 trial via **Main Menu > System Console > Edition and License > Start trial**. You can also create an account in the `Customer Portal <https://customers.mattermost.com/login/>`__, purchase a license key, and upload it to Mattermost via **System Console > Edition and License**.  
```

Comment:
For the 2nd sentence here, can we: 
1. Start the sentence with the intent (E.g. To purchase a license)  -- "You can also" makes it sound a bit like it's another way to get a trial, but really it seems to be talking about a separate case of when people are ready to buy. 
2. Say "Purchase online" instead of talking about the Customer Portal? 
  - I think people care most about the action rather than whether it's through a customer portal or not 
  - I think we should also consider pointing them to the pricing page or somewhere, since the link here lands on a sign up page with no information about the products they can buy